lib/index.js

```diff
@@ -607,43 +595,45 @@ function match(path, matcher) {
   throw new Error('wat is happening');
 }
 
-FSTree.prototype.filter = function(fn, context) {
-  return this.entries.filter(e => {
-    let entryPath = entryRelativePath(e);
+function filterByGlob(entryPath, cwd, files, include, exclude){
```

Comment:
can we rename this?  not all filters are globs; we filter by either string globs, `RegExp`s or matcher functions.